suite-common/wallet-config/src/types.ts

```diff
@@ -43,6 +43,18 @@ export const TREZOR_CONNECT_BACKENDS = [
     'blockfrost',
     'solana',
 ] as const;
+export const UNRELIABLE_TRANSACTION_COINS = [
```

Comment:
```suggestion
export const UNRELIABLE_TRANSACTION_NETWORKS = [
```

antifraud is enabled only for `networkType` === `ethereum`. Check `getIsPhishingTransaction` method

Reply:
This is very good point.

Reply:
🛑

Reply:
removed in favor of antifraud detenction, thanks